picATrip/urls.py

```diff
@@ -23,4 +24,7 @@
     path('', views.homepage, name='homepage'),
     path('about/', views.about, name='about'),
     path('register/', user_views.register, name='register'),
+    path('login/', auth_views.LoginView.as_view(template_name='users/login.html'), name='login'),
```

Comment:
Consider using the same base templates of the application, https://github.com/beyond-io/picATrip/blob/main/pickATrip_django_apps/templates/pickATrip_django_apps/base.html

Reply:
@Yarboa I am using this template inside 'login.html' by the command:
{% extends "pickATrip_django_apps/base.html"%}

Reply:
I did not see the navigation bar, is it working for you?

Reply:
I planned to create another PR and update the navigation bar with the new pages. I didn't want to add more changes because the navigation bar was created after I created this PR.

So I think you can merge this PR, and I will update the navigation bar in a new PR.